JellyfinPlayer tvOS/LatestMediaView.swift

```diff
@@ -42,15 +42,13 @@ struct LatestMediaView: View {
                 LazyHStack {
                     Spacer().frame(width: 45)
                     ForEach(items, id: \.id) { item in
-                        if item.type == "Series" || item.type == "Movie" {
```

Comment:
I don't really know why, but I had to remove this because Xcode started choking on this view and would not build. I tried commenting out various parts of this until I figured out that this was the culprit. 

If this filtering is important, could we move it out of this view and filter down the items list before it gets here? Let me know its importance and if there is another strategy that could be used.

Reply:
https://github.com/jellyfin/Swiftfin/pull/155/commits/a46ed4592539ad4e3fe1a71ae4136e342b8c7212

The same part of iOS has been modified in this commit. 
It has been modified to filter at the API call timing, so I don't think it matters if you just erase it.